onebusaway-android/src/main/java/org/onebusaway/android/ui/TripPlanFragment.java

```diff
@@ -70,6 +70,10 @@
 
 public class TripPlanFragment extends Fragment {
 
+    interface Host {
```

Comment:
Actually, better name would probably be `onTripRequestReady()` or something similar.